specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-10-01-preview/examples/CosmosDBBackupInformation.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2021-06-15",
```

Comment:
It seems the api-version in all samples are still using 2021-06-15, is it expected?